internal/engine/experiment/tor/tor.go

```diff
@@ -64,11 +62,11 @@ type TargetResults struct {
 }
 
 func registerExtensions(m *model.Measurement) {
-	oonidatamodel.ExtHTTP.AddTo(m)
-	oonidatamodel.ExtNetevents.AddTo(m)
-	oonidatamodel.ExtDNS.AddTo(m)
-	oonidatamodel.ExtTCPConnect.AddTo(m)
-	oonidatamodel.ExtTLSHandshake.AddTo(m)
+	archival.ExtHTTP.AddTo(m)
+	archival.ExtNetevents.AddTo(m)
+	archival.ExtDNS.AddTo(m)
+	archival.ExtTCPConnect.AddTo(m)
+	archival.ExtTLSHandshake.AddTo(m)
```

Comment:
This functionality of filling the extensions is also implemented in the `archival` package.